src/Sulu/Bundle/CoreBundle/Resources/config/request_analyzer.xml

```diff
@@ -26,13 +27,5 @@
             <tag name="sulu.context" context="website"/>
             <tag name="sulu.request_attributes" priority="0"/>
         </service>
-
-        <service id="sulu_core.webspace.request_listener"
```

Comment:
this listener calls the request analyzer on each request. i have moved this logic into a lazy initialization of the attributes.